src/main/resources/kobiton/ListDevices.py

```diff
@@ -0,0 +1,136 @@
+import json
+import re
+import urllib2
+
+kobiton_api_server = kobitonServer['url']
+username = kobitonServer['username']
+api_key = kobitonServer['password']
+
+# Return list in XebiaLabs
+devices = []
```

Comment:
The `devices` variable is a dictionary, not a list.
```
devices = {}
```
Please check and fix the below code